qutip/core/data/__init__.py

```diff
@@ -34,5 +35,20 @@
     (Dense, CSR, dense.from_csr, 1),
     (CSR, Dense, csr.from_dense, 1.4),
 ])
+to.register_alias('csr', CSR)
+to.register_alias('CSR', CSR)
+to.register_alias('Dense', Dense)
+to.register_alias('dense', Dense)
+
+
+import qutip.core.data.creator_util as creator_util
```

Comment:
Can just be `from . import creator_util` - but let's also call it `_creator_utils` (leading underscore and plural) instead to make it clear that it's completely private.  With the functions living inside `data.create`, things like Sphinx might still find it.